backend/zeno_backend/database/delete.py

```diff
@@ -197,6 +197,21 @@ def report_org(report_id: int, organization: Organization):
     )
 
 
+def dataset(project_uuid: str):
+    """Delete a dataset from a project.
+
+    Args:
+        project_uuid (str): id of the project to delete a dataset from.
+    """
+    with Database() as db:
+        db.execute(
+            sql.SQL("DROP TABLE IF EXISTS {} CASCADE;").format(
+                sql.Identifier(project_uuid)
+            )
+        )
+        db.commit()
```

Comment:
This gets called when one of the batch uploads fails. I guess what should really happen is just drop all of the project tables tbh, since they will get re-created by the dataset upload call again.